src/httpc.c

```diff
@@ -304,7 +304,7 @@ int
 httpc_set_body(struct httpc_request *req, const char *body, size_t size)
 {
 	ibuf_reset(&req->send);
-	char *chunk = ibuf_alloc(&req->send, size);
+	char *chunk = xibuf_alloc(&req->send, size);
 	if (chunk == NULL) {
```

Comment:
As far as I understand, we don't need the `if (chunk == NULL)` check after the change.

Reply:
Right, reverted:

```diff
diff --git a/src/httpc.c b/src/httpc.c
index c774be57af..a7abadf7b3 100644
--- a/src/httpc.c
+++ b/src/httpc.c
@@ -304,7 +304,7 @@ int
 httpc_set_body(struct httpc_request *req, const char *body, size_t size)
 {
 	ibuf_reset(&req->send);
-	char *chunk = xibuf_alloc(&req->send, size);
+	char *chunk = ibuf_alloc(&req->send, size);
 	if (chunk == NULL) {
 		diag_set(OutOfMemory, size, "ibuf", "http request body");
 		return -1;

```